### ISSUE TITLE
Resolvers in Traefik 101

### ISSUE THREAD

**Issue by aarontees** [ironicbadger/pms-wiki#104]
First of all, thank you for your Traefiki 101 guide—it got me up and running after a lot of searching. However, I ran into one hiccup, and that was not specifying the resolvers [here](https://github.com/ironicbadger/pms-wiki/blob/aed120a082d7113233e26e29e889a86b67ccdb26/docs/04-day-two/remote-access/traefik101.md?plain=1#L89) like you do in your infra repo. Once I included them I was up and running, so maybe they should be included in your bare minimum setup. Thanks again, love all you do!

**Comment by ironicbadger** [comment]
Good catch. I’ll update the guide to include resolvers or at least add a note about why they matter perhaps. Thanks for raising it.
